config/job.js

```diff
@@ -65,9 +65,14 @@ const SCHEMA_SETTINGS = Joi.object().optional();
 const SCHEMA_STEP = Joi.alternatives().try(SCHEMA_STEP_STRING, SCHEMA_STEP_OBJECT);
 const SCHEMA_STEPS = Joi.array().items(SCHEMA_STEP).min(1);
 const SCHEMA_TEMPLATE = Joi.string().regex(Regex.FULL_TEMPLATE_NAME);
+const SCHEMA_REQUIRES = Joi.alternatives().try(
+    Joi.array().items(Joi.string()),
```

Comment:
Do we need to make sure it's `~commit`, `~pr`, or doesn't start with `~`? Or we don't care to validate the value in requires?

Reply:
I think it will be hard to verify it, since it could also be `requires: jobName`, and `jobName` can be any string

Reply:
Correction: you're right. `jobName` is actually a regex, not just any string. So we can actually add some extra validation here.